tutorials/chemistry/05_Sampling_potential_energy_surfaces.ipynb

```diff
@@ -1,534 +1,609 @@
 {
```

Comment:
<p>Can you please re-run this cell with the most recent versions?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Ok I will.